### PR TITLE
Require proto rules to be loaded by buildifier

### PR DESCRIPTION
**Pull request by comius**
Because the rules are moved into the repository the loads become mandatory with Bazel@HEAD.

Add new warning for each rule and symbol that needs to be loaded. Buildifier doesn't handle well loading from different bzl files in a single warning.

Enable all proto rules warning by default.

### CODE REVIEW

**Comment by comius** [comment]
Hey @meteorcloudy, any idea why build is failing on protobuf on this PR, but not on daily builds?

File: buildifier/config/config_test.go [hunk 1]
```diff
@@ -260,9 +267,16 @@ func TestValidate(t *testing.T) {
 			"native-android",
 			"native-build",
 			"native-cc",
+			"native-cc-proto",
 			"native-java",
+			native-java-lite-proto",
```
Comment (meteorcloudy) [inline buildifier/config/config_test.go:272]: Missing a quote here.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: buildifier/config/config_test.go (hunk 1 above)
Comment (meteorcloudy) [reply buildifier/config/config_test.go:272]:
```
gentestmain: ParseFile("buildifier/config/config_test.go"): buildifier/config/config_test.go:272:26: string literal not terminated (and 10 more errors)
```

Comment (comius) [reply buildifier/config/config_test.go]: Ahhh thanks!! I was looking at protobuf warning and didn't see the error :)

**Review by meteorcloudy — APPROVED** [review]

**Comment by meteorcloudy** [comment]
@vladmos Please take a final look to merge it

**Review by vladmos — APPROVED** [review]

**Pull request merged**